### ISSUE TITLE
Speaker support

### ISSUE THREAD

**Issue by Merlinuf** [swttt/com.swttt.homekit#141]
Like sonos speakers, can you please add that?

**Comment by robertklep** [comment]
Can you make a screenshot from this page for that device: https://developer.athom.com/tools/devices

It will be helpful in adding support.

**Comment by Merlinuf** [comment]
Hi,

Dont know IF you got it, complained that it was to big, sent a bunch of pictures


Med vänlig hälsning,

Kristoffer

**Comment by Merlinuf** [comment]
Med vänlig hälsning,

Kristoffer

**Comment by robertklep** [comment]
Sorry, didn't get it :( Try uploading here (instead of e-mailing): https://github.com/swttt/com.swttt.homekit/issues/141

**Comment by Merlinuf** [comment]
![img_0554](https://user-images.githubusercontent.com/31373855/52857419-6933d180-3127-11e9-9b42-340b595c6123.jpeg)
![img_0555](https://user-images.githubusercontent.com/31373855/52857421-69cc6800-3127-11e9-8316-47062b5dce30.jpeg)
![img_0556](https://user-images.githubusercontent.com/31373855/52857422-69cc6800-3127-11e9-8f32-975de6e3c433.jpeg)
![img_0557](https://user-images.githubusercontent.com/31373855/52857423-69cc6800-3127-11e9-9168-69087a22a854.jpeg)
![img_0558](https://user-images.githubusercontent.com/31373855/52857424-69cc6800-3127-11e9-9e18-edeac48e5034.jpeg)

**Comment by Merlinuf** [comment]
I managed to do it now, hope it helps

**Comment by robertklep** [comment]
thanks!

**Comment by robertklep** [comment]
3.0.7 is released as a beta, should support speakers now.

**Comment by Merlinuf** [comment]
Yes now i can see them, but when i add them to home app on iPhone it don’t recognize it as speaker


Med vänlig hälsning,

Kristoffer

**Comment by robertklep** [comment]
That's something that @nhassink noticed during testing, but it doesn't make sense that it doesn't work. The Eve app (an alternative for the iOS Home app) apparently shows it correctly.

**Comment by Merlinuf** [comment]
Yes that is weird


Med vänlig hälsning,

Kristoffer

**Comment by FrankRinkens** [comment]
Sonos One isn't recognized in my config. Should this work? 

In dutch: Niet Geschikt, dit accessoire wordt mementeel niet ondersteund door de woning-app.

**Comment by robertklep** [comment]
@FrankRinkens that's the same problem that @Merlinuf (and also @nhassink) is having. Not yet sure what causes it. The [Eve app](https://itunes.apple.com/au/app/eve-for-homekit/id917695792) apparently does work.

**Comment by FrankRinkens** [comment]
Works in Eve app indeed. If you look into the Woning app config => If I look f.i. to Danalock => instellingen => Schakelaccessoire => I see accessoires number 7 pushing on ">" I see kamer Homey and everything is There except the Sonos One (see only 6 not 7). So the One is in my Favourite but not in that place " . don't know if this info will help?

![screen shot 2019-02-26 at 12 31 51](https://user-images.githubusercontent.com/32610925/53409988-c995ff00-39c2-11e9-810f-74ba16526b7a.png)
![screen shot 2019-02-26 at 12 32 06](https://user-images.githubusercontent.com/32610925/53409993-cef34980-39c2-11e9-9632-189e9d970556.png)

**Comment by robertklep** [comment]
It looks like only speakers that support Airplay 2 are supported by the Home app: https://support.apple.com/en-us/HT208724

My own Sonos Beam supports Airplay 2, but I don't know about other Sonos devices.

**Comment by FrankRinkens** [comment]
Installed another app "devices" => Sonos One also not visible. btw Sonos One supports Airplay 2

**Comment by robertklep** [comment]
@FrankRinkens and it shows up in the Settings page of the Homeykit app as "enabled"?

**Comment by FrankRinkens** [comment]
? settings tab says only settle time 120. apparaten tab => " One Keuken" in red verwijderen. so it is added. In the devices => dveloper athom.com => see previous attach => ready, available yes

**Comment by robertklep** [comment]
Try toggling it in Apparaten ("verwijderen" and after that "toevoegen") to see if it shows up in Homekit.

Otherwise, you'll have to wait until I get my Homey back from Athom (it got bricked during a firmware update) before I can debug this.

**Comment by FrankRinkens** [comment]
ok, maybe a miscommunication. It is showing up in Homekit. see attach. 
![sonos](https://user-images.githubusercontent.com/32610925/53411895-17f9cc80-39c8-11e9-9b83-a5f9f2671e9d.png)

**Comment by robertklep** [comment]
Okay, so we're still left with the "Niet geschikt". I have no idea why that happens at the moment.

**Comment by FrankRinkens** [comment]
The danalock shows up as a lock, the lamp as a bulb, but the sonos not as a speaker, seems like a wrong classification???

**Comment by robertklep** [comment]
It's added as a speaker, so not sure how else I would have to classify it ;)

**Comment by FrankRinkens** [comment]
;) I also don't know. Only thing I see is that at the Sonos One a "house" symbol is displayed and not a speaker. I also tried the Sonos Playbar and anoter One, same issue => "Niet geschikt"

**Comment by Moskvitch** [comment]
Hi, same here. A Sonos Connect can be controlled in the Eve - App On/Off, Volume, but in the Home App is shown as a Home icon with the correct Name as set in the Sonos app in Homey, with Not supported under.

**Comment by robertklep** [comment]
I wonder if this will only work (in the iOS Home app) for speakers that support Airplay _and_ support the HomeKit protocol themselves 🤔

**Comment by Patrikgolfer** [comment]
Same for me. My Yamaha WX-030 (airplay and Spotify connect) Speakers not supported i IOS Home app
![Speaker](https://user-images.githubusercontent.com/33980942/54473315-3e947180-47d6-11e9-96bb-eb0e7bdaf272.JPG)
